rel/i18n/emqx_schema.hocon

```diff
@@ -1633,10 +1633,15 @@ This inevitably limits the overall throughput of the system.
 session_ds_heartbeat_interval.label:
 """Durable session heartbeat interval"""
 session_ds_heartbeat_interval.desc:
-"""Durable sessions back up their state on disc every heartbeat interval.
+"""The interval at which a node with durable sessions reports its liveness.
 
-Heartbeat interval is also taken into account during garbage collection of expired sessions.
-It is added to the session expiry time as a safety margin when precise time of session termination is not precisely known."""
+In case of abnormal node shutdown, the node's last reported heartbeat time is used to
+determine the disconnection time of its durable sessions."""
+
+session_ds_commit_interval.label:
```

Comment:
Why do we need a different parameter?

Reply:
There are two different processes
* nodes bumping their liveness
* sessions committing their state

Reply:
The first is the optimization of the second.